tests/profiles/athena/test_athena_access_key.py

```diff
@@ -122,9 +173,19 @@ def test_athena_profile_args_overrides(
     """
     Tests that you can override the profile values for Athena.
     """
+    with patch(
+        "cosmos.profiles.athena.access_key.AthenaAccessKeyProfileMapping.get_temporary_credentials",
+        return_value=mock_assumed_credentials,
+    ):
+        yield
+
     profile_mapping = get_automatic_profile_mapping(
         mock_athena_conn.conn_id,
-        profile_args={"schema": "my_custom_schema", "database": "my_custom_db", "aws_session_token": "override_token"},
+        profile_args={
+            "schema": "my_custom_schema",
+            "database": "my_custom_db",
+            "aws_session_token": "override_token",
+        },
```

Comment:
Before, we were obfuscating the secrets and setting them in environment variables. This behaviour should continue for temporary credentials as well.